sdk/src/main/java/com/mapbox/maps/Style.kt

```diff
@@ -12,7 +12,7 @@ import java.nio.ByteBuffer
  * The general class to interact with Styles in the Modular Mapbox Maps SDK for Android.
  * It exposes the entry point for all methods related to the Style.
  * You cannot instantiate [Style] object directly, rather, you must obtain one
- * from the getStyleAsync() method on [MapboxMap].
```

Comment:
would be good to do an audit to all similar issues in the Style object, and make all the changes in a single PR

Reply:
I added many more fixes. Please take a second look.